its/ruling/src/test/resources/expected/python-S1751.json

```diff
@@ -1,5 +0,0 @@
-{
-'project:S2711.py':[
-6,
```

Comment:
Here we are removing a TRUE POSITIVE on a check that is not supposed to be the target of what we are doing. For me this is a regression and thus, there is something fishy going on.

Reply:
As discussed, this was raised on the following code:
```
class MyClass(object):
    ' doc string '
    while True:
        return False
```
that although being correct syntactically, it's not correct semantically, so we don't create a CFG for it.

Reply:
Fair enough : we only raise issue on valid python code.